### DISCUSSION TITLE
Announcing the winners of RAGHack 2024!

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#171] [Announcements]
Hello everyone, thank you for your patience. On behalf of the RAGHack team, I want to thank you all for participating in this livestream series and hackathon. We enjoyed seeing everyone apply their learnings for their projects!

Without further ado... here are the winners for each category:
- Best Overall: DocAssistant.Charty
- Best in PostgreSQL: Football-Analysis-Copilot
- Best in Azure SQL: UniChatbot
- Best in Cosmos DB: Discord Community Agent
- Best in Azure AI Search: Manufacturing Support Bot
- Best in Python: StoryWeave
- Best in .NET: ContosoTravelAgency
- Best in Java: Paper Mentor AI
- Best in JavaScript/TypeScript: Learning Path Certifications Builder
- Best in Azure AI Studio: MyFitnessBuddy - AI powered Fitness Assistant

Thank you all for participating! If you want to check out any of these projects in more depth, follow this link to our published winners list on the MS Developer Community Blog! 

[aka.ms/raghack/winners-blog](https://aka.ms/raghack/winners-blog)

**Comment by dubscode** [comment]
Congratulations to the winners! Big thank you to the MS team for putting this together, was a great experience.

**Comment by Liquid72** [comment]
How could I claim the prize?

**Reply by multispark** [reply]
Keep an eye on your email inbox that you used to register for the hack!

**Comment by jeremiah-louis** [comment]
@multispark when would we receive our badges of participation

**Reply by marcel-paalvast** [reply]
@multispark still waiting on mine. I did fill in the form back when I received it.

**Reply by multispark** [reply]
The team is working on distributing badges, we are having some technical difficulties. Thank you for your patience.
